custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_v2.json

```diff
@@ -1377,6 +1377,43 @@
               }
             },
             "property_value": 183
+          },
+          {
+            "type": "or",
```

Comment:
I think this is right. It includes them (which is my understanding of a filter, let me know if thats wrong) when the person is NOT `registered_status=not_registered` OR NOT `migrated=yes`, which is to say it includes them if either of the two does not match the exclusion condition. My read of the spec is that both of those properties need to be met to exclude them but it is a bit ambiguous